src/Tester/OrleansConfigurationForSilosStopUnitTests.xml

```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
you do need a separate new XML file for that.
There is a virtual base method in the UnitTestSiloHost class that can be overwritten to tweak the config object.
Look here: https://github.com/ReubenBond/orleans/commit/3001fa7765b0ea30ac5f0a1a17ab3a99802d8b6e